llvm/test/Verifier/RemoveDI/blockbyref.ll

```diff
@@ -0,0 +1,18 @@
+; RUN: llvm-as -disable-output <%s 2>&1| FileCheck %s
```

Comment:
nit: don't need `<`? and while you're there can you add a space between `1` and `|`?

EDIT: this is in a few of the other tests too. Were these copied and modified from existing tests? Feel free to ignore this nit